commands/templates/svc/manifest-public.yml

```
@@ -30,25 +27,28 @@ sidecars:
     port: 443
     image: public.ecr.aws/uktrade/nginx-reverse-proxy:latest
     variables:
-      SERVER: localhost:{% if ipfilter %}8000{% else %}8080{% endif %}
+      SERVER: localhost:8000
 
-{% if ipfilter %}
   ipfilter:
     port: 8000
     image: public.ecr.aws/uktrade/ip-filter:latest
     variables:
       PORT: 8000
-      EMAIL_NAME: 'The Department for International Trade WebOps team'
-      EMAIL: test@test.test
-      LOG_LEVEL: DEBUG
+      EMAIL_NAME: DIT
```

We should probably go with DBT here.

```suggestion
      EMAIL_NAME: DBT
```